Robust.Client/Input/InputManager.cs

```diff
@@ -523,13 +523,16 @@ public void LoadKeyFile(ResPath file, bool defaultRegistration, bool userData =
             {
                 var baseKeyRegs = _serialization.Read<KeyBindingRegistration[]>(BaseKeyRegsNode, notNullableOverride: true);
 
+
                 foreach (var reg in baseKeyRegs)
                 {
+                    var invalid = false;
+
                     if (reg.Type != KeyBindingType.Command && !NetworkBindMap.FunctionExists(reg.Function.FunctionName))
                     {
                         Logger.ErrorS("input", "Key function in {0} does not exist: '{1}'", file,
                             reg.Function);
```

Comment:
This is an expected condition going forward so shouldn't be reported as an error.
```suggestion
                        Logger.DebugS("input", "Key function in {0} does not exist: '{1}'.", file,
                            reg.Function);
```